### ISSUE TITLE
Discussion: Why Google Analytics, and why high priority?

### ISSUE THREAD

**Issue by tilda** [thedaviddias/Front-End-Checklist#405]
I understand that analytics might be important for y'all, but I personally think that GA isn't really that much of a necessary tool considering:
- how much data it collects
- the size of the scripts you need to include to load it in (thus increasing load time by a significant amount)
- it's blocked by adblockers (at least by default in uBlock Origin, unsure about others)
- etc, etc

There might be something I'm missing here, but why is it needed?

**Comment by sandip2224** [comment]
I suppose it's popular because firstly, it's free for everyone, and secondly, data collection is automated without manual intervention. What do you think about it?

**Comment by akrantz01** [comment]
Personally, I'm not a fan of Google Analytics because of the amount of data they collect about you and your site. It's free because you aren't paying with money, you're paying with data instead. The amount of data that you give them is significantly more than the amount you give them. They also don't necessarily show the most accurate statistics if you hit a certain limit for page views under the free tier. 

As such, I don't think Google Analytics itself is required, but it would be nice to still have some checkbox for analytics since people like seeing how their website is doing. It also shouldn't be a high priority since not everyone needs it, maybe reduce it to a medium or even low priority.
